packages/components/src/components/floating-menu/floating-menu.js

```diff
@@ -279,10 +321,29 @@ class FloatingMenu extends mixin(
       this._place();
       // IE11 puts focus on elements with `.focus()`, even ones without `tabindex` attribute
       if (!this.element.hasAttribute(this.options.attribAvoidFocusOnOpen)) {
-        (
-          this.element.querySelector(this.options.selectorPrimaryFocus) ||
-          this.element
-        ).focus();
+        const primaryFocusNode = this.element.querySelector(
+          this.options.selectorPrimaryFocus
+        );
+        const focusableNode = this.element.querySelector(
+          settings.selectorTabbable
+        );
+
+        if (primaryFocusNode) {
+          primaryFocusNode.focus();
+        } else if (focusableNode) {
+          focusableNode.focus();
+        } else {
+          this.element.focus();
+          const elementTabindex = this.element.getAttribute('tabindex');
+          if (__DEV__) {
+            // @question the demo that doesn't have interactive elements is throwing this warning
+            //    despite that i have tabindex="0" and the statement in the first parameter is "false"
```

Comment:
Would you want to elaborate this...? Is this something that can be demonstrated with our dev env with your latest commit?

Reply:
@asudoh yes you can see this with the latest commit.  If you click on the second demo you'll get a warning in the console but the HBS file has `tabindex="0"` on that demo so it should not be throwing the warning.

Reply:
Thank you for explaining @elizabethsjudd! Given you have `!elementTabindex || elementTabindex === '-1'` and you seem to have expected `tabindex="0"` not emit the warning,  did you expect `!elementTabindex` to catch `tabindex="0"` condition? Note that any attributes values are `string` (`DOMString`) or `null`.

Reply:
wow... yea that's my bad.  I totally wasn't thinking about the fact that 0 = false (must have needed more coffee that day)

Reply:
@asudoh I made an update to specifically look for `null` and the warning is still being thrown in the console.  I console logged the first parameter of the warning message and it returned `false` so I don't understand why the warning is being thrown.

Reply:
Did you mean `elementTabindex === '0' || elementTabindex === '-1'`, given the your `@question` comment seems to have described that you wanted to catch `tabindex="0"` and `tabindex="-1"` scenarios? My apologies if you actually were meant for missing `tabindex` attribute as your latest comment here seems to have alluded to.

Reply:
@asudoh, I want to throw the error if the tooltip does not have `tabindex` at all or that it's set to `-1`.  In this use-case the tooltip is not focusable yet there is not a focusable element within it so the HTML should have `tabindex="0"` (or any number other than `-1`) based on our demos.